### PR TITLE
Fix and Workaround for crash of leaving Edit View to Early

### LINKED ISSUE #90: [BUG] Application crashes when changing menu immediately after click save on profile settings
**Describe the bug**
If you were to save your account profile settings and immediately go to a different menu of the application before waiting for the menu that you profile edits have saved, the application will crash.

**To Reproduce**
1. Go to edit your account profile
2. Save your profile settings
3.  Before the menu that your profile edits have saved, go to a different menu of the program (any menu works as far as I can tell)
4. Crash will occur

**Expected behavior**
Either it will enter the menu and still save the settings or lock me out from entering any menu before it saves.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 22621.160

**Additional context**
Video and crash log of the problem will be listed.

[Video](https://user-images.githubusercontent.com/95731834/175396408-b4c5d261-08fa-439b-b3c3-d7bdc23775a1.mp4)

[Crash Log](https://github.com/Founntain/osuplayer/files/8970835/20220623_204511.txt)

### PR DESCRIPTION
**Pull request by Founntain**
This will fix #90

### CODE REVIEW

File: OsuPlayer/Views/EditUserView.axaml.cs
Comment (Cesan) [inline OsuPlayer/Views/EditUserView.axaml.cs:209]: Won't this fail if the ViewModel is checked to be null before and we check a property of it here?

Comment (Founntain) [reply OsuPlayer/Views/EditUserView.axaml.cs]: Yes this is true, I forgot to add it into my commit. I pushed it

**Comment by Founntain** [comment]
@Cesan I updated it. If you want to see the explaination, check the resolved comment

**Review by Cesan — APPROVED** [review]
Looks good

**Comment by Founntain** [comment]
Then we wait for approval in the issue, if the pre-release fixed itm than we can merge it.

**Comment by sajidbhuiya** [comment]
Just had a look at the pre-release and seems to have been fixed!

**Pull request merged**
